packages/okta-react/test/e2e/harness/src/index.js

```diff
@@ -16,5 +16,12 @@ import './index.css';
 import App from './App';
 import registerServiceWorker from './registerServiceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+// To perform end-to-end PKCE flow we must be configured on both ends: when the login is initiated, and on the callback
+// The login page is loaded with a query param. This will select a unique callback url
+// On the callback load we detect PKCE by inspecting the pathname
```

Comment:
Is this information devs using this SDK would need to know/implement?  If not, why is this test different than our sample code?

Reply:
our samples use environment variable, you would have to restart the server to switch PKCE on or off.

Reply:
This method I am using here is more brittle, and would not be recommended for users. But to use environment variables I would need to run 2 completely separate test cycles and run the webpack again.